### PR TITLE
Dataset test

### PR DESCRIPTION
**Pull request by RafaelPignata622**
I add the test for the __init__.py inside the dataset

### CODE REVIEW

File: .github/workflows/unittests.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install -U coverage pytest flake8 pydocstyle
+        python -m pip install -U coverage pytest flake8 pydocstyle check-manifest pytest-cov
```
Comment (vcristiani) [inline .github/workflows/unittests.yml:22]: Check this because git actions raise an error.

**Review by vcristiani — COMMENTED** [review]
Looks good. Perhaps we should add a test that tests the ValueError that was added.

File: .github/workflows/unittests.yml (hunk 1 above)
Comment (RafaelPignata622) [reply .github/workflows/unittests.yml:22]: Done

**Review by ornemarioni — APPROVED** [review]
All looks ok

**Pull request merged**